shuup/core/models/_orders.py

```diff
@@ -863,6 +864,11 @@ def create_shipment(self, product_quantities, supplier=None, shipment=None):
         if not supplier:
             supplier = shipment.supplier
 
+        if not supplier.modules:
```

Comment:
you already have the exception inside `supplier.ship_products`, no need to have it here.

Reply:
Fixed.